src/Jira/Api.php

```diff
@@ -347,6 +348,33 @@ public function getVersions($projectKey)
     }
 
     /**
+     * Helper method to find a specific version based on the name of the version.
+     *
+     * @param string $projectKey Project Key
+     * @param string $name The version name to match on
+     *
+     * @return int|null VersionId on match or null when there is no match
+     */
+    public function findVersionByName($projectKey, $name)
```

Comment:
This method is meant to be public. (and is not used internally in the lib yet) 
I'm using it in a project to search the corresponding release based on a git tag :)